src/main/js/add-item.js

```diff
@@ -38,50 +37,49 @@ $.when(getItems()).done(function (data) {
     }
 
     function getCopyFromValue() {
-      return $('input[type="text"][name="from"]', "#createItem").val();
+      return document.querySelector(
+        '#createItem input[type="text"][name="from"]',
+      ).value;
     }
 
     function isItemNameEmpty() {
-      var itemName = $('input[name="name"]', "#createItem").val();
-      return itemName === "" ? true : false;
+      var itemName = document.querySelector(
+        '#createItem input[name="name"]',
+      ).value;
+      return itemName.trim() === "";
     }
 
     function getFieldValidationStatus(fieldId) {
-      return $("#" + fieldId).data("valid");
+      return document.querySelector("#" + fieldId).dataset.valid === "true";
     }
 
     function setFieldValidationStatus(fieldId, status) {
-      $("#" + fieldId).data("valid", status);
+      document.querySelector("#" + fieldId).dataset.valid = status;
     }
 
     function activateValidationMessage(messageId, context, message) {
       if (message !== undefined && message !== "") {
-        $(messageId, context).text("» " + message);
+        document.querySelector(context + " " + messageId).textContent =
+          "» " + message;
       }
       cleanValidationMessages(context);
-      $(messageId).removeClass("input-message-disabled");
-      enableSubmit(false);
+      document
+        .querySelector(messageId)
+        .classList.remove("input-message-disabled");
+      refreshSubmitButtonState();
     }
 
     function cleanValidationMessages(context) {
-      $(context)
-        .find(".input-validation-message")
-        .addClass("input-message-disabled");
+      document
+        .querySelectorAll(context + " .input-validation-message")
+        .forEach((element) => element.classList.add("input-message-disabled"));
     }
 
-    function enableSubmit(status) {
-      var btn = $(".bottom-sticker-inner button[type=submit]");
-      if (status === true) {
-        if (btn.hasClass("disabled")) {
-          btn.removeClass("disabled");
-          btn.prop("disabled", false);
-        }
-      } else {
-        if (!btn.hasClass("disabled")) {
-          btn.addClass("disabled");
-          btn.prop("disabled", true);
-        }
-      }
+    function refreshSubmitButtonState() {
```

Comment:
Rather than accept a true/false parameter lets just have the function handle whether the button should be valid or not.